policyengine_us/variables/gov/states/md/tax/income/agi/subtractions/md_two_income_subtraction.py

```diff
@@ -26,9 +26,9 @@ def formula(tax_unit, period, parameters):
         couple_gross_income = tax_unit.sum(
             where(is_head | is_spouse, gross_income, 0)
         )
-        head_frac = where(
-            couple_gross_income > 0, head_gross_income / couple_gross_income, 1
-        )
+        head_frac = np.ones_like(couple_gross_income)
```

Comment:
```suggestion
        # Compute the head's share of the couple's cross income.
        # Use a mask rather than where to avoid a divide-by-zero warning. Default to one.
        head_frac = np.ones_like(couple_gross_income)
```